### PR TITLE
Adding posts for existing events page

### PR DESCRIPTION
**Pull request by bmorris3**
I just noticed there's an existing framework for rendering a static events on the OpenAstronomy website [see here](https://openastronomy.org/news.html). This is a first attempt to populate that page with more events.  cc @bsipocz @dpshelio

### CODE REVIEW

**Comment by bsipocz** [comment]
Good discovery. I wouldn't mix gsoc with the meetings though, and split between past and future meetings, but all of these can be left for the future.

Also see the question here https://github.com/OpenAstronomy/openastronomy.github.io/issues/163#issuecomment-389243845 about which meetings to list. I'm inclined to include most of this format, but probably we want to have a discussion about it.

File: _posts/2018-09-24-dotastro.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: 2018 Septemeber 24-27 - .Astronomy X
+
+date: 2018-09-24
+---
+
+The tenth .Astronomy meeting will take place in Baltimore MD from September 
+24-27, 2018. [see here for more info](https://www.dotastronomy.com/home-ten/).
```
[the remark this answers is not in the document]
Comment (dpshelio) [reply _posts/2018-09-24-dotastro.md]: I would just put the link on the title:
```markdown
[The tenth .Astronomy meeting](https://www.dotastronomy.com/home-ten/) will take place in Baltimore MD from September 
24-27, 2018.
```

**Comment by bmorris3** [comment]
Done, @bsipocz / @dpshelio.

**Comment by bsipocz** [comment]
Thanks @bmorris3!

**Pull request merged**